.evergreen/scaffold_atlas.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import annotations
+
+import json
+import logging
+import os
+from pathlib import Path
+from typing import Any, Union
+
+from pymongo import MongoClient, database
+from pymongo.results import InsertManyResult
+
+logging.basicConfig()
+logger = logging.getLogger(__file__)
+logger.setLevel(logging.DEBUG if os.environ.get("DEBUG") else logging.INFO)
+
+DATABASE_NAME = os.environ.get("DATABASE")
+CONN_STRING = os.environ.get("CONN_STRING")
+REPO_NAME = os.environ.get("REPO_NAME")
+DIR = os.environ.get("DIR")
+TARGET_DIR = os.environ.get("TARGET_DIR")
+DB_PATH = "database"
+
+
+def upload_data(client: database, filename: Path) -> None:
+    """Take the supplied file contents and upload the data collection
+
+    :param database: The database linking to the Mongo Atlas client
+    :param filename: Collection with json contents
+    """
+    loaded_collection: Union[list[dict[str, Any]], dict[str, Any]]
```

Comment:
Do we need all of the explicit type hints for these variables? Not sure they add clarity here.

Reply:
I'll push back here and say we keep it. It's a personal preference to maintain typing whenever possible, especially in the case that there are multiple different types.